yt_dlp/extractor/ninegag.py

```diff
@@ -32,22 +34,40 @@ class NineGagIE(InfoExtractor):
         # HTML escaped title
         'url': 'https://9gag.com/gag/av5nvyb',
         'only_matching': True,
+    }, {
+        # Non Anonymous Uploader
+        'url': 'https://9gag.com/gag/ajgp66G',
+        'info_dict': {
+            'id': 'ajgp66G',
+            'ext': 'webm',
+            'title': 'Master Shifu! Or Splinter! You decide:',
+            'upload_date': '20220806',
+            'timestamp': 1659803411,
+            'thumbnail': 'https://img-9gag-fun.9cache.com/photo/ajgp66G_460s.jpg',
+            'categories': ['Funny'],
+            'tags': ['Funny'],
+            'duration': 26,
+            'like_count': int,
+            'dislike_count': int,
+            'comment_count': int,
+            'uploader': 'Peter Klaus',
+            'uploader_id': 'peterklaus12',
+            'uploader_url': 'https://9gag.com/u/peterklaus12',
+        }
     }]
 
     def _real_extract(self, url):
         post_id = self._match_id(url)
-        post = self._download_json(
+        post = traverse_obj(self._download_json(
             'https://9gag.com/v1/post', post_id, query={
                 'id': post_id
-            })['data']['post']
+            }), ('data', 'post'))
```

Comment:
Revert this. It didn't use traverse_obj/try_get because the entire extraction will fail anyway if this fails.